django part 1/1. understanding

```diff
@@ -0,0 +1,60 @@
+1.  it's where you will create the ruleset of the db (how the db will look like)
+    it's in charge on how the db will look like (rules, set-up)
+
+2.  the views.py is where you will put all the logic and the functions. it's in charge of all the requests.
+
+3.  urls.py is the place that will tell the computer where to go when a certain word is put into the search bar.
+    it's in charge of the stream of info, and to direct where each info is stored.
```

Comment:
Idea is right - the terminology should be a bit different.
The thing we put in the 'search bar' (which is not really a search bar necessarily) is called the URL.
so this file is responsible to tell which functions handle which URLs.
But again, the idea was definitely correct.